source/renderer/app/i18n/locales/en-US.json

```diff
@@ -428,15 +428,15 @@
   "test.environment.selfnodeLabel": "!!!Selfnode vx",
   "test.environment.stagingLabel": "Staging",
   "test.environment.testnetLabel": "Testnet",
-  "wallet.add.dialog.create.description": "Create a new wallet",
+  "wallet.add.dialog.create.description": "Create a wallet for the balance check",
```

Comment:
Revert this line to what it was @tomothespian

Reply:
@darko-mijic Done!